apps/ledger-live-desktop/src/renderer/live-common-setup.ts

```diff
@@ -27,3 +29,26 @@ registerTransportModule({
   },
   disconnect: () => Promise.resolve(),
 });
+
+registerTransportModule({
+  id: "vault-transport",
+  open: (id: string) => {
+    const prefixID = "vault-transport:";
+    if (!id.startsWith(prefixID)) return;
+    // FIXME how to restore initial device mode value?
+    setDeviceMode("polling");
```

Comment:
in future, the device mode should be provided by the transport module because it seems related to the transport used anyway. i think you can go with a hack that setDeviceMode it back in the usb transport module's open() but it's something we will need to track somewhere ( cc @juan-cortes @alexandremgo )

Reply:
Yep agree it feels weird to set it back in the `open` of an another transport, but I can think of a better solution  :/

Reply:
@gre just pushed this commit because i realized it was not working with a real device. It seems we actually never open a transport with `id=ipc` so the module was never picked up. May be it changed because I remember testing toggling on and off vault transport when I initially worked on that (weeks ago now).